specification/cognitiveservices/data-plane/Face/readme.md

```diff
@@ -21,6 +21,13 @@ These settings apply only when `--tag=release_1_0` is specified on the command l
 input-file: stable/v1.0/Face.json
 ```
 
+### Release 1.0-preview
```

Comment:
I'm a little confused on this versioning.  Seems like a v1.0-preview would come before a stable v1.0 which already exists.  Can you please clarify?

Reply:
The case for this feature is a little special as it is approved by API review board to go into Face API stable v1.0 after going GA. As there is no next target stable version, we figured v1.0-preview would best convey the intention.

Reply:
Thanks for clarifying.  Can I diff this against the v1.0 stable to see the delta?

Reply:
OK comparing against v1.0 does the trick.

Does this mean that this version will replace the current v1.0 stable?

Reply:
When the diff (Person Directory feature) eventually goes into GA, yes.